### ISSUE TITLE
Using pth_and_npth_single_file causes KiBot to crash

### ISSUE THREAD

**Issue by RRvW** [INTI-CMNB/KiBot#26]
When I export with the following config snippet:
```yaml
  - name: "Drill files"
    type: excellon
    dir: pcb_output
    options:
      pth_and_npth_single_file: True
```
I get the following error:
```
Traceback (most recent call last):
  File "/usr/bin/kibot", line 16, in <module>
    main()
  File "/usr/lib/python3/dist-packages/kibot/__main__.py", line 319, in main
    generate_outputs(outputs, args.target, args.invert_sel, args.skip_pre)
  File "/usr/lib/python3/dist-packages/kibot/kiplot.py", line 262, in generate_outputs
    out.run(get_output_dir(out.dir), board)
  File "/usr/lib/python3/dist-packages/kibot/out_base.py", line 55, in run
    self.options.run(output_dir, board)
  File "/usr/lib/python3/dist-packages/kibot/out_any_drill.py", line 106, in run
    os.rename(k_file, file)
FileNotFoundError: [Errno 2] No such file or directory: '<dir>/HEARP_PCB-PTH.drl' -> '<dir>/HEARP_PCB-PTH_drill.drl'
```
Using Kibot 0.7.0 from the docker image `setsoft/kicad_auto`

**Comment by set-soft** [comment]
This should be fixed by 577713efeb581750d50e3f5030e88a2c110996a3

Can you try it?

**Comment by RRvW** [comment]
Works great, thanks a lot!
